statsmodels/regression/linear_model.py

```diff
@@ -1112,7 +1139,8 @@ def __init__(self, model, params, normalized_cov_params=None, scale=1.,
                 # TODO: warn or not?
             self.get_robustcov_results(cov_type=cov_type, use_self=True,
                                        use_t=use_t, **cov_kwds)
-
+        for key in kwargs:
+            setattr(self, key, kwargs[key])
```

Comment:
is there a difference to just doing `self.__dict__update(kwargs)` which, IIRC, we use sometimes?